cmd/create.go

```diff
@@ -189,6 +189,33 @@ A full list of component types that can be deployed is available using: 'odo cat
 	},
 }
 
+/*
+ parseCreateCmdArgs returns
+ 1. image name
+ 2. component type i.e, builder image name
+ 3. component name default value is component type else the user requested component name
+ 4. component version which is by default latest else version passed with builder image name
+*/
```

Comment:
you shouldn't use `/*` for multiple line comments with functions. Please change it to `//`

Reply:
Ack